From fedb67176acb91087a68167673c568de43a5768c Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Wed, 18 Apr 2018 16:59:55 +0800
Subject: [PATCH 3/6] driver: ethernet: instead of cycle_t with u64

According to upstream commit a5a1d1c2914b("clocksource: Use
a plain u64 instead of cycle_t"), cycle_t had been removed.
So, use u64 to replace it.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_emacps.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_emacps.c b/drivers/net/ethernet/xilinx/xilinx_emacps.c
index 0e45cf4..275a080 100644
--- a/drivers/net/ethernet/xilinx/xilinx_emacps.c
+++ b/drivers/net/ethernet/xilinx/xilinx_emacps.c
@@ -1056,7 +1056,7 @@ static inline void xemacps_ptp_read(struct net_local *lp,
  * @cc: Cyclecounter structure
  * Return: Hw time stamp
  */
-static cycle_t xemacps_read_clock(const struct cyclecounter *cc)
+static u64 xemacps_read_clock(const struct cyclecounter *cc)
 {
 	struct net_local *lp = container_of(cc, struct net_local, cc);
 	u64 stamp;
-- 
1.8.5.2.233.g932f7e4

